### ISSUE TITLE
Limite para geração do arquivo pdf

### ISSUE THREAD

**Issue by anapaulaweiss** [cesarvr/pdf-generator#119]
dependendo o tamanho do html ao tentar gerar para pdf o plugin aborta o aplicativo, 160kb já aborta.

**Comment by cesarvr** [comment]
Hi @anapaulaweiss,

For what I can understand the plugin is basically crashing for content above 160KB. Unfortunately I cannot reproduce this error.  I wrote [this project example](https://github.com/cesarvr/pdf-generator-example) that maybe can provide you with some guidance, here is quick demo: 


![ezgif com-video-to-gif](https://user-images.githubusercontent.com/3899337/81609735-4d71b680-93d0-11ea-8b61-52abd36e0643.gif)

> As you can see in those two example the plugin is able to convert HTML content of considerable size.

**Comment by anapaulaweiss** [comment]
qual tamanho aproximado desse html do seu exemplo?

**Comment by cesarvr** [comment]
[default.pdf](https://github.com/cesarvr/pdf-generator/files/4612065/default.pdf)
[my-pdf.pdf](https://github.com/cesarvr/pdf-generator/files/4612067/my-pdf.pdf)
First one is 870KB
The Second one is 4.9 MB

**Comment by anapaulaweiss** [comment]
realmente, fiz um teste aqui, e são 160kb só do texto html, mas possui muitas imagens e acaba passando de 7MB, nesse caso qual seria a sugestão para esses arquivos com muitas imagens e que acabam ficando muito grande?

**Comment by cesarvr** [comment]
Sorry I don't speak Portuguese, but google translator tells me that you are asking something about:  "We can assume the PDF files are that big because of the images" -- the answer will be yes, the PDF file acts as a container (or zip file) that encompass all kind of assets such as words, images, videos, etc.

**Comment by anapaulaweiss** [comment]
of course, sorry. Would you have a solution that you would recommend for this type of larger file?

**Comment by anapaulaweiss** [comment]
there is no error, just abort the application, I realized that for some cell phones with the same file size it did not abort the application and for another yes, it seems to be related to the memory of the cell
